core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -79,6 +79,8 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8:
     platform: aws
+  cucushift-installer-rehearse-aws-ipi-baselinecaps-none:
```

Comment:
Would you mind to sort them with alphanumeric order ?

Reply:
Resloved